src/util/removeEmpty.spec.js

```diff
@@ -0,0 +1,25 @@
+import assert from 'assert';
+import removeEmpty from './removeEmpty';
+
+describe('removeEmpty', () => {
+    context('with no empty values', () => {
```

Comment:
As you commented, I merged the `context` into the `it`. So now there are no `context`s.